src/heltour.js

```diff
@@ -166,7 +166,33 @@ function getPrivateURL(heltourConfig, page, user){
         'page': page,
         'user': user
     };
+    return fetchJSONandHandleErrors(request);
+}
+
+function assignAlternate(heltourConfig, round, team, board, player){
+    var request = heltourRequest(heltourConfig, "assign_alternate");
+    request.method = "POST";
+    request.bodyParameters = {
+        "league": heltourConfig.leagueTag,
+        "round": round,
+        "team": team,
+        "board": board,
+        "player": player
+    };
+    return fetchJSONandHandleErrors(request);
+}
+
+function getLeagueModerators(heltourConfig){
+    var request = heltourRequest(heltourConfig, "get_league_moderators");
+    request.parameters = {
+        "league": heltourConfig.leagueTag
+    };
+    return fetchJSONandHandleErrors(request).then(function(json){
+        return json["moderators"];
+    });
+}
 
+function fetchJSONandHandleErrors(request){
```

Comment:
Nice, I like this. We can (eventually) add in handling for non-200 errors and not trying to parse json if there is no body. Like the 405 error.